other/botCodeStore.ts

```diff
@@ -39,6 +39,13 @@ export function getBots() {
   return { ...STORE };
 }
 
+export function getBotUsername(botId: string) {
+  if (!STORE[botId]) {
+    return "";
```

Comment:
Если что-то не должно происходить, выкидывай ошибку, не замалчивай её.

```suggestion
    throw new Error(`bot with botId=${botId} not found in the botCodeStore`);
```